src/rust/inetstack/protocols/layer4/udp/tests.rs

```diff
@@ -451,87 +420,6 @@ fn udp_loop2_ping_pong() -> Result<()> {
     Ok(())
 }
 
-//======================================================================================================================
```

Comment:
Let's move these three tests to the integration tests. We should create a tests/rust/udp-test directory and move these tests to a bind/mod.rs. Then we will not need a separate directory for windows and linux in nettest, which is not designed to handle error cases.

Reply:
Done.